applications/ehr_query_llm/README.md

```diff
@@ -0,0 +1,43 @@
+# Generative AI Application on Holoscan integrating with FHIR Services
+
+This reference application demonstrates how Holoscan application developers can build healthcare Gen AI applications that easily interact with EHR systems without the need for per-hospital customization, accelerating the development of AI tools to help improve clinician efficiency and quality of patient care.
+
+**_Note_**:
+This application is NOT for clinical use.
+
+<img src="resources/EHR_LLM_Arch.png" alt="isolated" width="800"/>
+
+As shown in the architecture diagram above, there are three main components in this application
```

Comment:
@MMelQin please modify the diagram above to accurately describe the retrieval and the generative component of the RAG pipeline. We do not have a Context retrieval op and a query formation op, the name and the diagram is misleading for developers that can be lead to think that we incapsulated the functionality in Holoscan operators

Reply:
It is an architecture diagram, not a high level design diagram. I'll see what to change.